intents.json

```diff
@@ -259,13 +269,26 @@
                 "Personally, I find my creator 'Shashin' very inspiring. I might not be very fair though... since half of me was also built by 'Saugat', 'Sabal' and 'Neema'."
             ]
         },
+        {
```

Comment:
Remove the greetreply tag as well 
Since `greeting` tag already exists

Reply:
greetreply removed